test/unit/spec/MfaVerify_spec.js

```diff
@@ -2855,7 +2858,15 @@ function (Okta,
             return Expect.waitForSpyCall(webauthn.getAssertion, test);
           })
           .then(function (test) {
-            expect(test.form.el('o-form-error-html').length).toBe(1);
+            return Expect.wait(() => {
```

Comment:
Since we're moving toward a pattern where the `errorBox` can be populated with dynamic messages (custom views, events, etc) - I wonder if we should add a shared `Expect` method called `waitForErrorBox` that will handle this for us.

Reply:
I doubt it will introduce some confusion because we have `waitForFormError` today.
The reason it doesn't here is because the error was added manually rather from Courage hence the error doesn't have expected class.
Basically `.okta-form-infobox-error` v.s. `.okta-infobox-error`.

Maybe we shall when it appears in many places.

Reply:
Sounds good!

Reply:
with this change we have `return Expect.waitForSpyCall(webauthn.getAssertion, test);` followed by another wait `return Expect.wait(() => {`
Do we need both or can we get rid of the first one and then just assert that it was called (without a wait block)
```
  return Expect.waitForFormErrors(test.form, test);
})
.then(function (test) {
  expect(webauthn.getAssertion).toHaveBeenCalled();
  expect(test.form.errorBox().length).toBe(1);
  ...
```